### ISSUE TITLE
[Feature Request] View AUR PKGBUILD's from right click menu

### ISSUE THREAD

**Issue by Strit** [manjaro/pamac#87]
A feature I'd like to see, is the ability to right click on a package from AUR when searching, and be given the option to View/Edit the PKGBUILD. If the PKGBUILD gets edited it could use the edited version when installing the package.

The last part might be hard to do, but at least let me view a PKGBUILD so I know what the package is gonna do, without the need to visit aur.archlinux.org.

Thank you. :)

**Comment by Strit** [comment]
With the now built in AUR support, I feel this is much more important now, just need to add the ability to EDIT the PKGBUILD aswell, since it might be needed for some AUR packages.

Please consider doing this, either with calling ther default GUI text editor or opening a terminal window with nano in it.

**Comment by polarathene** [comment]
I'd really like this, I have several packages that are frustrating to update with Pamac due to the way the PKGBUILD is setup(source URLs that get latest version but verify against a specific checksum are fragile). All I need to do is open up the location the package files were downloaded to and get the new md5checksum value then update the PKGBUILD during install to use that new md5sum.

I've had similar issues with some packages where the dependencies need to be tweaked or something else.

Is this feature going to get some love? Anything in particular making it difficult to implement?

**Comment by guinux** [comment]
Yes ,editing support would requires to add a  big piece of code to parse the modified PKGBUILD to have new deps.
For now Pamac only used AUR rpc interface to have those info.

**Comment by polarathene** [comment]
You can't just have the PKGBUILD passed to an external editor like default text editor, save to temporary location(/tmp) and then substitute that modified PKGBUILD at build time in place of the AUR one?

**Comment by guinux** [comment]
No it's more complicated.
An example: if you edit the PKGBUILD to add a dep which is a package from AUR, pamac have to know it must build and install this new dep before.
